lib/api/controllers/serverController.js

```diff
@@ -299,9 +299,12 @@ class ServerController extends NativeController {
   async openapi (request) {
     const format = request.getString('format', 'json');
     const scope = request.getString('scope', 'all');
+    const exclude = request.getString('exclude', '').split(',').map(e => e.trim()).filter(e => e);
```

Comment:
You should use [request.getArray](https://docs.kuzzle.io/core/2/framework/classes/kuzzle-request/get-array/) instead